tests/components/RangeInput.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import { ComponentMeta } from '@storybook/react';
+import { RangeInput, RangeInputProps } from '../../src/components/Filters/RangeInput';
+import { AnswersHeadlessContext, Matcher, SelectableFilter } from '@yext/answers-headless-react';
+import { generateMockedHeadless } from '../__fixtures__/answers-headless';
+import { FiltersContext, FiltersContextType } from '../../src/components/Filters/FiltersContext';
+import { FilterGroupContext, FilterGroupContextType } from '../../src/components/Filters/FilterGroupContext';
+import { userEvent, within } from '@storybook/testing-library';
+
+const meta: ComponentMeta<typeof RangeInput> = {
+  title: 'RangeInput',
+  component: RangeInput,
+};
+
+export default meta;
+
+const selectableFilter: SelectableFilter = {
+  selected: true,
+  fieldId: '123',
+  matcher: Matcher.Equals,
+  value: 'test'
+};
+
+const filterContextValue: FiltersContextType = {
+  selectFilter: () => null,
+  applyFilters: () => null,
+  filters: []
+};
+
+const filterContextValueDisabled: FiltersContextType = {
+  selectFilter: () => null,
+  applyFilters: () => null,
+  filters: [selectableFilter]
+};
+
+const filterGroupContextValue: FilterGroupContextType = {
+  searchValue: '',
+  fieldId: '123',
+  setSearchValue: () => null,
+  getCollapseProps: null,
+  getToggleProps: null,
+  isExpanded: null,
+  isOptionsDisabled: null,
+  setIsOptionsDisabled: () => null
+};
+
+export const Primary = (args: RangeInputProps) => {
+  return (
+    <AnswersHeadlessContext.Provider value={generateMockedHeadless()}>
+      <FilterGroupContext.Provider value={filterGroupContextValue}>
+        <FiltersContext.Provider value={filterContextValue}>
+          <RangeInput {...args}/>
+        </FiltersContext.Provider>
+      </FilterGroupContext.Provider>
+    </AnswersHeadlessContext.Provider>
+  );
+};
+
+export const Disabled = ((args: RangeInputProps) => {
+  return (
+    <AnswersHeadlessContext.Provider value={generateMockedHeadless()}>
+      <FilterGroupContext.Provider value={filterGroupContextValue}>
+        <FiltersContext.Provider value={filterContextValueDisabled}>
+          <RangeInput {...args}/>
+        </FiltersContext.Provider>
+      </FilterGroupContext.Provider>
+    </AnswersHeadlessContext.Provider>
+  );
+// eslint-disable-next-line no-extra-bind
+}).bind({});
```

Comment:
why is this bind call needed? I think we can remove it since the play steps below is directly attach to `Disabled`?

Reply:
I needed the bind call since storybook would not let me call play without it.  In the more updated push, I didn't need this anymore, so it's now removed!